src/zmb/features/languages/generated/client/client.json

```diff
@@ -1071,7 +1071,7 @@
         "sysAdHome.user.audit.deactiv.reasonNotEmp": "No longer an employee",
         "sysAdHome.user.audit.deactivation.subtitle": "This will revoke {name}’s ability to login and access the system. The account can be reactivated at a later date.",
         "sysAdHome.user.audit.deactivation.title": "Deactivate {name}?",
-        "sysAdHome.user.audit.form.error": "Please complete all mandatory fields",
+        "sysAdHome.user.audit.form.error": "A reason is required for deactivating this user",
```

Comment:
This should be `A reason is required for {action} this user` and action should be `deactivating/reactivating`